sample_config.yml

```diff
@@ -0,0 +1,64 @@
+
+# The GitHub organization to host your GitHub classroom
+# Create a new organization on GitHub and change this to the name you gave it
+organization: earth-analytics-edu
+
+# Course directory. Your config.yml file should be in this directory and all
```

Comment:
i think we can delete these lines. getting a duplicate key error at the CLI

Reply:
looks like i need to put several paths into the config file not just the course name path?

```
(abc-dev) CIRES-EL-LM-020:my-course leahwasser$ abc-assignment-template sdfs
Loading configuration from config.yml
Creating new directory for template repos at /Users/me/awesome-course/assignment_repos
Traceback (most recent call last):
  File "/Users/leahwasser/miniconda3/envs/abc-dev/bin/abc-assignment-template", line 11, in <module>
    load_entry_point('abc-classroom', 'console_scripts', 'abc-assignment-template')()
  File "/Users/leahwasser/Documents/github/0-python/abc-classroom/abcclassroom/__main__.py", line 440, in assignment_template
    config, assignment, args.mode
  File "/Users/leahwasser/Documents/github/0-python/abc-classroom/abcclassroom/template.py", line 31, in create_template_dir
    os.mkdir(parent_path)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/me/awesome-course/assignment_repos'
```

Reply:
Did you already run `abc-quickstart`, and it added another course_directory? otherwise, I am confused about how we ended up with duplicates in the config.

Reply:
the course directory is defined twice @kcranston  in your sample config. i removed nathan's and copied yours like you suggested i do! i was trying to fix things and then just looked at the file and saw the duplicate keys. it's in this PR.

https://github.com/earthlab/abc-classroom/pull/91/files?file-filters%5B%5D=.py&file-filters%5B%5D=.yml#diff-b595fa0c7e24765055981d1fca476e85R8

https://github.com/earthlab/abc-classroom/pull/91/files?file-filters%5B%5D=.py&file-filters%5B%5D=.yml#diff-b595fa0c7e24765055981d1fca476e85R21

what i'm going to do is create a new branch and clean stuff up. it might just be easier than a ton of comments from me again.